src/CommunityVoices/Public/js/create-slide.js

```diff
@@ -11,8 +11,9 @@ if ($('#slide_text').length) { // if youre editing slide
     var current_text = 'Quote goes here',
         current_attr = 'Attribution',
         current_image = 10,
```

Comment:
I am suspicious about this line.  What about when there isn't an image 10?  Then do we just show a broken image link?  I feel like it would be better to show nothing.

Reply:
Yes, it would just show a broken image link in that case. I thought that this would be okay since at least still indicates that there will/should be an image there, but if you think it's better to just show nothing, I'll make that  change.

Reply:
Note also that showing nothing will cause the quote and attribution to be in the wrong place until an image is selected. If we really don't want the broken image link, would it be possible to have some sort of generic/default image to use?

Reply:
Unfortunately, it does not look like there is a default image currently stored in the file system.  If you want to just make a black box, you can make a `div` with `width` and `height` set to whatever you think is a good ratio (`30vw` for 30% of the width, etc.) and that is a quick way to do the black box approach.

Could also make the image a div that has text within it saying "Image goes here" or something.  I'm not really concerned about how it is done, I just think that we shouldn't rely on anything existing in the database.  Otherwise, what do we show when we are making the first slide of a new database?

Reply:
I think the solution I've come up with is good. It's a simple but I think it looks nice enough and conveys what it needs to.